src/components/Hamburger.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import '../App.css';
+import styled, { css } from 'styled-components'
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+
+const StyledMenu = styled.nav`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #EFFFFA;
+  transform: ${({ open }) => open ? 'translateX(0)' : 'translateX(-100%)'};
+  height: 80vh;
+  text-align: left;
+  border-radius: 15px;
+  padding: 1.5rem;
+  position: absolute;
+  top: 0;
+  left: 0;
+  transition: transform 0.3s ease-in-out;
+
+  @media (max-width: 576px) {
+      width: 100%;
+    }
+
+  a {
+    font-size: 2rem;
+    text-transform: uppercase;
+    padding: 2rem 0;
+    font-weight: bold;
+    letter-spacing: 0.1rem;
+    color: #0D0C1D;
+    text-decoration: none;
+    transition: color 0.3s linear;
+
+    @media (max-width: 576px) {
+      font-size: 1.5rem;
+      text-align: center;
+    }
+
+    &:hover {
+      color: #343078;
+    }
+  }
```

Comment:
better to put this whole thing out of this file to a different file (maybe StyledMenu.style.js) and import it. 
Same for the StyleBurger